overlays/haskell.nix

```diff
@@ -11,6 +11,8 @@ final: prev: {
         # overlays.
         defaultModules = [];
 
+        # TODO: doc etc
+        extraPkgconfigMappings = {};
```

Comment:
I thought making this an attribute of `haskell-nix` and leveraging overlays would be the most composable route to propagating this to where it needed to go. Sadly, it didn't propagate to cross-compilation.

Reply:
This should probably be part of the project-common module.